package.json

```diff
@@ -79,13 +79,33 @@
     "request": "^2.81.0",
     "semver": "^5.3.0",
     "simple-git": "^1.65.0",
-    "sinon": "^2.2.0",
+    "sinon": "^7.1.1",
     "tarjan-graph": "^0.3.0",
     "tmp": "0.0.26",
     "upath": "^0.1.7",
     "uuid4": "^1.0.0",
     "when": "^3.7.7",
-    "wreck": "^12.2.2"
+    "wreck": "^12.2.2",
+    "npm-which": "^3.0.1",
```

Comment:
We should move a lot of these dependencies as `devDependencies` since the task is only going to be used by folks working on it. These should not be installed for folks using it to work on their theme.

Reply:
Agreed

Reply:
@mattolson @junedkazi  Moved all the package that I could, appveyor require the test related packages under the "dependencies"

Reply:
We can update appveyor.yml file to run dev dependencies as well. But We should not be installing packages which are not required for stencil cli to function on end user machines. @mattolson thoughts ?

https://github.com/bigcommerce/stencil-cli/blob/master/appveyor.yml#L10